apps/website/src/pages/docs/avatar.mdx

```diff
@@ -13,73 +13,53 @@ import * as AllExamples from 'examples';
 
 <LiveExample src='Avatar.sizes.tsx'>
   <AllExamples.AvatarSizesExample client:load />
 </LiveExample>
 
-- `small`
-- `medium`
-- `large`
-- `x-large`
-
-Which size to use is at your discretion. Determine which size is best for your context based on available space.
-
-### Initials
-
-If a user does not upload a photo, the user’s first & last initials are displayed. You can display a maximum of 2 letters and a minimum of 1 letter. In the off chance there are user accounts without any name associated, you can display the first letter of the user’s email address.
-
-<LiveExample src='Avatar.initials.tsx'>
-  <AllExamples.AvatarInitialsExample client:load />
-</LiveExample>
-
-### Icon
-
-An icon may be display instead of initials, typically used by organizations instead of a person.
-
-<LiveExample src='Avatar.icon.tsx'>
-  <AllExamples.AvatarIconExample client:load />
-</LiveExample>
+By making use of the `size` prop and selecting one of the predefined values mentioned above, you can easily set the size of the `Avatar` component.
 
-### Picture
+### Status indicators
 
-If a user uploads a photo we display their photo instead of their initials.
+Status indicators are indicators that display in the bottom right of the avatar. Statuses can be set using the `status` prop. The available statuses are: `online`, `away`, `busy` and `offline`.
 
-<LiveExample src='Avatar.picture.tsx'>
-  <AllExamples.AvatarPictureExample client:load />
+<LiveExample src='Avatar.statuses.tsx'>
+  <AllExamples.AvatarStatusesExample client:load />
 </LiveExample>
 
-### Status indicators
+We suggest not using statuses on the small size avatar.
 
-You may want to indicate the online status of a user. Status indicators display in the bottom right of the avatar. We suggest not using statuses on the small size avatar. The available statuses are:
+## Variants
 
-- `online`
-- `away`
-- `busy`
-- `offline`
+An avatar can be customized to show initials, icons, and pictures.
 
-<LiveExample src='Avatar.statuses.tsx'>
-  <AllExamples.AvatarStatusesExample client:load />
+<LiveExample src='Avatar.main.tsx'>
+  <AllExamples.AvatarMainExample client:load />
 </LiveExample>
 
-## Avatar group
+When no profile picture is uploaded, an avatar with the user's initials is shown. It can display one or two letters, or the first letter of their email if no name is provided. You may use an icon instead. Once a photo is uploaded, it replaces the initials/icon.
 
-Avatars can be displayed in a list to represent a group of people. This visual treatment reinforces the fact that these people work together and/or share similar permissions to access/view/edit a file or a project. For example, you might display the list of people who previously edited an item, are all collaborating on a specific activity, etc. To save space, consider the "Stacked" option below.
```

Comment:
To confirm, we don't want to keep this?

Reply:
We can keep some of it

Suggested wording:
> Avatars can be displayed in a list to represent a group of people using `<AvatarGroup>` component. This visual treatment reinforces the fact that these people work together and/or share similar permissions to access/view/edit a file or a project.
>
> By default, the avatars will overlay each other to save space. This can be turned off by setting `stacked={false}` if needed.